lib/dor/workflow_exception.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 module Dor
-  class WorkflowException < ::RuntimeError
```

Comment:
just curious, what does the removal of the "::" do?

Reply:
@peetucket Adding the explicit `::` tells Ruby to resolve a constant in the top-level namespace. It's typically added when trying to avoid picking up, say, `MyModule::Errors::RuntimeError` instead of `RuntimeError`. Removing `::` lets Ruby run its usual namespace resolution routines, going up the ancestor chain.

Reply:
In this case, I find it noisy and potentially confusing for newer Rubyists.